src/otx/engine/adaptive_bs/adaptive_bs_api.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-"""Algorithm to find a proper batch size which is fit to current GPU device for tasks using mmcv."""
+"""Algorithm to find a proper batch size which is fit to current GPU device for tasks using mmcv."""  #
```

Comment:
@eunwoosh I have a question, is this module used only for tasks using mmcv?

Reply:
No, I just copied it from otx 1.X. It works with other tasks.

Reply:
If so, can I remove it?

Reply:
yes sure. Thanks :)